tools/code-generation/generator/src/main/resources/com/amazonaws/util/awsclientgenerator/velocity/cpp/StreamRequestSource.vm

```diff
@@ -76,6 +76,32 @@ Aws::Http::HeaderValueCollection ${typeInfo.className}::GetRequestSpecificHeader
 
 }
 #end
+
```

Comment:
ah this is is what is causing the whitespace change, put it inside the `#if`. a empty line in  a ` vm` file will result in a new line in the code that is generated.